guides/components/index.md

```diff
@@ -1,85 +1,79 @@
+    <div class="avatar is-active" title="Avatar de Tomster">T</div>
   </aside>
   <section>
     <h4 class="username">
       Tomster
-      <span class="local-time">their local time is 4:56pm</span>
+      <span class="local-time">heure locale : 16:56</span>
     </h4>
 
     <p>
-      Hey Zoey, have you had a chance to look at the EmberConf brainstorming doc
-      I sent you?
+      Hey Zoey, as-tu eu l'occasion de regarder le document
+      de brainstorming pour l'EmberConf que je t'ai envoyé ?
     </p>
   </section>
 
   <aside class="current-user">
-    <div class="avatar" title="Zoey's avatar">Z</div>
+    <div class="avatar" title="Avatar de Zoey">Z</div>
   </aside>
   <section>
     <h4 class="username">Zoey</h4>
 
-    <p>Hey!</p>
+    <p>Hey !</p>
 
     <p>
-      I love the ideas! I'm really excited about where this year's EmberConf is
-      going, I'm sure it's going to be the best one yet. Some quick notes:
+      J'adore les idées ! Je suis vraiment enthousiaste de voir la
+      direction que prend l'EmberConf cette année, je suis sûre que ce
+      sera la meilleure de toutes.
+      Quelques notes rapides :
     </p>
 
     <ul>
       <li>
-        Definitely agree that we should double the coffee budget this year (it
-        really is impressive how much we go through!)
+        Tout à fait d'accord sur le fait qu'on devrait doubler le budget
+        café cette année (c'est vraiment impressionnant tout ce qu'on
+        boit !)
       </li>
       <li>
-        A blimp would definitely make the venue very easy to find, but I think
-        it might be a bit out of our budget. Maybe we could rent some spotlights
-        instead?
+        Un dirigeable rendrait certainement le lieu très facile à trouver,
+        mais je pense que c'est peut-être un peu hors budget. On
+        pourrait peut-être louer des projecteurs à la place ?
       </li>
       <li>
-        We absolutely will need more hamster wheels, last year's line was
-        <em>way</em> too long. Will get on that now before rental season hits
-        its peak.
+        On a absolument besoin de plus de roues pour hamster, l'année dernière
+        la file d'attente était <em>beaucoup</em> trop longue. Je vais m'occuper
+        de ça maintenant, avant que la saison des locations soit à son pic.
       </li>
     </ul>
 
-    <p>Let me know when you've nailed down the dates!</p>
+    <p>Dis-moi quand tu auras fixé les dates !</p>
   </section>
 
   <form>
     <label for="message">Message</label>
     <input id="message" />
     <button type="submit">
-      Send
+      Envoyer
     </button>
   </form>
 </div>
 ```
 
-You can _serve_ the app by running `ember s` in your terminal, which will make
-the local copy of your application available to view in your web browser.
+Vous pouvez "servir" l'app en exécutant `ember s` dans votre terminal, ce qui rendra la copie locale de votre app disponible pour votre navigateur web.
```

Comment:
Why not simply saying:
> Vous pouvez démarrer l'app ...
> Vous pouvez exécuter l'app ...

Reply:
It's just that English put the word _serve_ in italics, like a term to learn, so I didn't want to change it. "Servir" is the correct translation and can be used in French as well "Le rôle d'un serveur est de servir".